djangocms_version_locking/helpers.py

```diff
@@ -67,3 +68,34 @@ def placeholder_content_is_unlocked(placeholder, user):
     """
     content = placeholder.source
     return content_is_unlocked(content, user)
+
+
+def create_version_lock(version, user):
+    """
+    Create a version lock
+    """
+    VersionLock.objects.create(
+        version=version,
+        created_by=user
+    )
+
+
+def remove_version_lock(version):
+    """
+    Delete a version lock, handles when there are none available.
+    """
+    VersionLock.objects.filter(version=version).delete()
```

Comment:
Return the value of `delete()`, or return True if at least 1 object has been deleted